CONTRIBUTING.md

```diff
@@ -424,6 +514,9 @@ Here's some more content.
 On initial page load, the first tab ("<your title here>" in the example above)
 will be the one displayed.
 
+> **Note:** You can’t nest tabs within tabs.
```

Comment:
```suggestion
> **Note:** You can’t nest navtabs within navtabs.
```

Suggesting using the command name because I am correcting this syntax in plugin docs. 

![image](https://user-images.githubusercontent.com/3756245/88809828-2b3f3780-d17a-11ea-99bf-61b6e9b5cdc0.png) from https://github.com/Kong/docs.konghq.com/commit/24378ad662de5780feca9a16708e688f734c8ce7#diff-e7bfaf0d7282472697dbb2f05400a498L163


Do you want such an indent and gray sidebar for the note?

![image](https://user-images.githubusercontent.com/3756245/88809562-e5826f00-d179-11ea-9c8e-27dd28fa66ce.png)

Reply:
Re: note formatting, that's the way GitHub renders notes, so I don't have any issues with it.